src/IdeasWorkshop/ThemeManager.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace AppBundle\IdeasWorkshop;
+
+use AppBundle\Entity\IdeasWorkshop\Theme;
+use AppBundle\Storage\ImageStorage;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+
+class ThemeManager
+{
+    public function __construct(ImageStorage $imageStorage)
+    {
+        $this->imageStorage = $imageStorage;
+    }
+
+    public function saveImage(Theme $theme): void
```

Comment:
This image management is repeated in multiple classes like
https://github.com/EnMarche/en-marche.fr/blob/master/src/Biography/BiographyManager.php
https://github.com/EnMarche/en-marche.fr/blob/master/src/Mooc/MoocManager.php
It would be nice to create a dedicated class.

Reply:
IMO it should be in another PR, isolated from admin changes, but yes definitely.